CHANGELOG.asciidoc

```diff
@@ -40,6 +40,14 @@ https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
 - Fix spooling to disk blocking infinitely if the lock file can not be acquired. {pull}15338[15338]
 - Fix `metricbeat test output` with an ipv6 ES host in the output.hosts. {pull}15368[15368]
 - Fix `convert` processor conversion of string to integer with leading zeros. {issue}15513[15513] {pull}15557[15557]
+- Fix missing output in dockerlogbeat {pull}15719[15719]
```

Comment:
we are just releasing dockerlogbeat with 7.6. I think we can remove this line.

Reply:
I'll remove the line.